### ISSUE TITLE
Type conversion from BigInt to Bytes

### ISSUE THREAD

**Issue by schmidsi** [graphprotocol/docs#223]
https://thegraph.com/docs/en/developing/assemblyscript-api/#type-conversions-reference

**Comment by azf20** [comment]
@schmidsi are you saying this needs to be added to the docs?

**Comment by schmidsi** [comment]
@azf20 Yes. Or maybe even better support it in TS. Sorry for not better speccing this out since I was focusing on something else but wanted to leave the note here.

I ended up with something like:
```ts
Bytes.fromI32(event.params.tokenId.toI32()
```

But to transform a BigInt to I32 is removing info and should be avoided. Better would be `Bytes.fromBigInt` or `BigInt.toBytes()`. But for now, we could document the code above.
